cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -301,6 +302,11 @@ public KafkaAssemblyOperator(Vertx vertx, PlatformFeaturesAvailability pfa,
                 .compose(state -> state.entityOperatorDeployment())
                 .compose(state -> state.entityOperatorReady())
 
+                .compose(state -> state.getExporterOperatorDescription())
+                .compose(state -> state.exporterOperatorServiceAccount())
+                .compose(state -> state.exporterOperatorDeployment())
+                .compose(state -> state.exporterOperatorReady())
+
```

Comment:
We have service template, we seem to have it in the Model. But where do we create the service?